src/Console/Command.php

```diff
@@ -277,6 +277,8 @@ public function executeCommand($command, array $args = [], ConsoleIo $io = null)
         try {
             return $command->run($args, $io);
         } catch (StopException $e) {
+            $io->error($e->getMessage());
```

Comment:
Couldn't this result in duplicate error messages in existing use cases?

Reply:
Yes, that is what's happening and why I think there is an issue for this to be fixed in 3.x now.
Still sad that we can't use the exception message for what it is supposed to be used for.

Reply:
I didn't expect people to directly throw `StopException` from their command classes. Using the `abort()` helper removes the need for additional imports and results in more terse code.